apps/website/.vuepress/config.js

```diff
@@ -82,7 +82,8 @@ module.exports = {
     [
       'robots',
       {
-        allowAll: true,
+        disallowAll: true,
```

Comment:
This works for now, but do we have an idea for how to toggle this based on the deploy environment? Something for the future at least.

Reply:
I don't think it will work. It will block the Algolia crawler and that's not what we want. 

I'll dig around for some environment flags/switches to see what I can come up with.